File: problemas/iniciante/1172/1172.scala
object Main {   
  def main(args: Array[String]): Unit = {
    for(i <- 0 to 9){
        val x = io.StdIn.readLine().toInt

        if(x > 0){
            printf("X[%d] = %d\n", i, x)
        }else{
            printf("X[%d] = 1\n", i)
        }
    }
  }
}

File: problemas/iniciante/1172/1172.rs
use std::io;

fn main() {
    for i in 0..10 {
        let mut input = String::new();
        io::stdin().read_line(&mut input);
        let numero: i32 = input.trim().parse().unwrap();

        if numero <= 0 {
            println!("X[{}] = 1", i);
        } else {
            println!("X[{}] = {}", i, numero);
        }
    }
}
